.travis.yml

```diff
@@ -0,0 +1,63 @@
+language: rust
+
+env:
+  global:
+    - CRATE_NAME=easage
+    - FEATURES=clap
+    # 'rust:' statements in lint stage also need to be modified
+    - NIGHTLY=nightly-2018-01-01
```

Comment:
Does travis fail for some reason if we use stable?
Maintaining consistency across CI and local systems is important.

Reply:
$NIGHTLY global variable here is for deciding what version cargo should use (expands to `cargo +nightly-2018-01-01` in `lint` stages. This is because clippy needs nightly, and since we use nightly anyway, might as well use nightly version of rustfmt while at it.

What actually decides which rust version travis uses is the `rust: nightly-2018-01-01` statement, which doesn't take in variables. Hence the comment that you need to modify `NIGHTLY=` as well as `rust: nightly-2018-01-01`

Stages where the `rust:` statement is omitted are defaulting to stable rust, (e.g.`stage: build`)
You need to set nightly explicitly like we've done in `lint` stage, or travis assumes stable rust for everything.